src/dtx/dtx_resync.c

```diff
@@ -245,6 +245,13 @@ dtx_status_handle_one(struct ds_cont_child *cont, struct dtx_entry *dte, daos_un
 		 * let's commit the DTX globally.
 		 */
 		D_GOTO(out, rc = DSHR_NEED_COMMIT);
+	case -DER_OOG:
```

Comment:
I am not quite sure if DER_OOG can always retry (for example when the target is not in Primary group will return DER_OOG, will it cause dead loop for some cases if always retry?), is there a chance to refresh pool map?

Reply:
If the rank/target is not excluded, then since there is yield() for each cycle retry, then it will not always OOG as long as the group membership can be initialized finally; otherwise if it has already been excluded, then seems no matter even if the excluded one fall into dead loop?